s3tests_boto3/functional/test_s3_neofs.py

```diff
@@ -598,3 +599,25 @@ def test_multipart_upload():
     _check_content_using_range(key, bucket_name, data, 1000000)
     _check_content_using_range(key, bucket_name, data, 10000000)
     client.delete_object(Bucket=bucket_name, Key=key)
+
+
+@attr(resource='bucket')
+@attr(operation='bucket workflow')
```

Comment:
I think `operation` should describe what a test is doing, then I suggest add more information like: `create and delete bucket`